data/sql_setup/prepare_schedule_b.sql

```diff
@@ -82,43 +314,30 @@ $$ language plpgsql;
 create or replace function ofec_sched_b_delete_update_queues() returns trigger as $$
 declare
     start_year int = TG_ARGV[0]::int;
-    timestamp timestamp = current_timestamp;
-    two_year_transaction_period smallint;
     view_row fec_fitem_sched_b_vw%ROWTYPE;
 begin
-    if tg_op = 'DELETE' then
-        select into view_row * from fec_fitem_sched_b_vw where sub_id = old.sub_id;
-
-        -- Check to see if the resultset returned anything from the view.  If
-        -- it did not, skip the processing of the record, otherwise we'll end
-        -- up with a record full of NULL values.
-        -- "FOUND" is a PL/pgSQL boolean variable set to false initially in
-        -- any PL/pgSQL function and reset whenever certain statements are
-        -- run, e.g., a "SELECT INTO..." statement.  For more information,
-        -- visit here:
-        -- https://www.postgresql.org/docs/current/static/plpgsql-statements.html#PLPGSQL-STATEMENTS-DIAGNOSTICS
-        if FOUND then
-            two_year_transaction_period = cast(get_cycle(view_row.rpt_yr) as smallint);
-
-            if two_year_transaction_period >= start_year then
-                delete from ofec_sched_b_queue_old where sub_id = view_row.sub_id;
-                insert into ofec_sched_b_queue_old values (view_row.*, timestamp, two_year_transaction_period);
-            end if;
-        end if;
+    select into view_row * from fec_fitem_sched_b_vw where sub_id = old.sub_id;
 
+    -- Check to see if the resultset returned anything from the view.  If
+    -- it did not, skip the processing of the record, otherwise we'll end
+    -- up with a record full of NULL values.
+    -- "FOUND" is a PL/pgSQL boolean variable set to false initially in
+    -- any PL/pgSQL function and reset whenever certain statements are
+    -- run, e.g., a "SELECT INTO..." statement.  For more information,
+    -- visit here:
+    -- https://www.postgresql.org/docs/current/static/plpgsql-statements.html#PLPGSQL-STATEMENTS-DIAGNOSTICS
+    if FOUND then
+        if view_row.election_cycle >= start_year then
+            delete from ofec_sched_b_queue_old where sub_id = view_row.sub_id;
+            insert into ofec_sched_b_queue_old values (view_row.*, current_timestamp, view_row.election_cycle);
```

Comment:
Same note as in schedule A, we shouldn't have to deal with the `old` queue tables anymore either, right?